website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
Really? Someone can have permission to remove contributors without having permission to remove them? I'd like to audit this ...